bridge/slack/slack.go

```diff
@@ -18,22 +19,52 @@ type Bslack struct {
 	mh           *matterhook.Client
 	sc           *slack.Client
 	rtm          *slack.RTM
```

Comment:
The `Usergroups` fields is used nowhere. There are also not public uses of the `Users` and `UseChannelID` as this struct implements a generic `bridge.Bridge` interface and has no use for publicly-exported fields.